frontend/src/routes/HomePage.js

```diff
@@ -96,6 +111,12 @@ class HomePage extends React.Component<{
                                     {auth.isAuthenticated &&
                                         this.renderButton('My Profile', '/users')}
                                 </div>
+                                <div>
+                                    {this.renderStartButton(
+                                        'Get Started',
```

Comment:
Could we make this appear on the same line as the other buttons?

Reply:
Sure! Make the buttons on the same line now:
<img width="1177" alt="Screen Shot 2021-04-29 at 7 08 03 PM" src="https://user-images.githubusercontent.com/34461466/116641314-4f09f680-a921-11eb-8dba-50664f3d113a.png">

Reply:
Maybe we rename "Get started" to "Read Quickstart"? This is so it's clearer that that button goes to the documentation (technically, logging in / signing up is also a way to "get started")

Reply:
thanks for the advice. The button has been renamed to "QUICKSTART"